Tests/SentryTests/Profiling/SentryProfilerSwiftTests.swift

```diff
@@ -0,0 +1,320 @@
+import Sentry
+import XCTest
+
+#if os(iOS) || os(macOS) || targetEnvironment(macCatalyst)
+class SentryProfilerSwiftTests: XCTestCase {
```

Comment:
It's hard to review this code, as it was moved from the `HubTests`. It's hard to see which code is new and which one was just moved. Easier for the reviewer would be to have the code still in `HubTests` and move the code then later on in a different PR, or move the tests before this PR in an extra PR.

Reply:
Sorry about that, I also did some refactoring after moving it out. I'll point you to the new functionality in some review comments.